dotcom-rendering/stories/generated/LayoutReadme.stories.jsx

```diff
@@ -0,0 +1,28 @@
+
+import { css } from '@emotion/react';
+
+const ReadMe = () => (
+	<section css={css`
+		padding: 1rem;
+		& h1, p{
+			margin-bottom: 0.5rem;
+		}
+		`}>
+		<h1 css={css`font-size: 1.5em; font-weight: 400;`}>Readme</h1>
+		<p>
+			The stories in this directory are automatically generated.
+		</p>
+		<p>
+			To add new format variations to test, please edit the list in `get-stories.js`.
+		</p>
+	</section>
+);
+
+// eslint-disable-next-line import/no-default-export
+export default {
+	title: 'Components/Layout/Format Variations',
+	component: ReadMe,
+	chromatic: { disableSnapshot: true },
+};
```

Comment:
[And it shows up nicely](https://637e40d1bc73bf3f604394b9-wboekubbsr.chromatic.com/?path=/story/components-layout-format-variations--readme)!